mysql/import.sql

```diff
@@ -15,7 +15,11 @@ INSERT INTO users (
 ('Fifth','User','fifth@user.com','fifthUser','$2b$10$6Hlgf70PI6lZT5ZP7RRNQuLfRcG3wx.PsAcOu983TgFEp/bhDYKwa','https://res.cloudinary.com/breads/image/upload/v1613539580/tortilla_fvnmgn.jpg','2020-03-27 11:46:36');
 
 -- READINGS
+<<<<<<< HEAD
```

Comment:
This syntax is from a merge conflict. Check out the current `mysql/import.sql` and update these lines